tock/tock/templates/base.html

```diff
@@ -37,6 +37,35 @@
     ga('set', 'forceSSL', true);
     ga('send', 'pageview');
   </script>
+  <script>
```

Comment:
i still don't like having this much logic in a template but trying to move it over to a JS file  led to some annoying race conditions.